presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcHandleResolver.java

```diff
@@ -59,12 +60,24 @@ public boolean canHandle(ConnectorOutputTableHandle tableHandle)
     }
 
     @Override
+    public boolean canHandle(ConnectorTableLayoutHandle handle)
+    {
+        return handle instanceof JdbcTableLayoutHandle;
```

Comment:
@electrum would it be better if I just got connectorId like this? :
`handle.getTable().getConnectorId()`